### ISSUE TITLE
White line above taskbar when maximized

### ISSUE THREAD

**Issue by IcyFoxe** [jurplel/qView#727]
**Environment:**
- OS: Windows 10
- qView version: 6.1

**Information:**
There is a white line at the bottom of the window, right above the taskbar, when maximized. It's a bit distracting especially with the dark background.

![image](https://github.com/user-attachments/assets/6aa6a3e8-2826-4a3a-a3d4-7dbe022976e9)

On a side note, I discovered this app today, and I already like it. It's what I've been looking for - a simple, customizable and performant app that supports modern formats. Thank you and keep up the good work!

**Comment by jdpurcell** [comment]
This might be better on the nightly builds (from GitHub Actions) since they use a newer Qt version, not because Qt truly fixed it but it seems more likely to glitch to black pixels instead of white at least when I was testing. Qt has issues in general with fractional scaling ([relevant bug report](https://bugreports.qt.io/browse/QTBUG-126990)) and 175% is probably the worst. It causes other problems like this in the settings dialog where some controls may have missing borders.

![image](https://github.com/user-attachments/assets/89bdad69-3c77-4d07-9cb2-4bc2142ce73b)

**Comment by IcyFoxe** [comment]
Yup yup 175% is exactly my desktop scaling, because it's the perfect size for my 4k monitor.
